### PR TITLE
Count kuviz for public map quota

### PR DESCRIPTION
**Pull request by gonzaloriestra**
Related to https://github.com/CartoDB/cartodb/issues/15261

### CODE REVIEW

**Comment by gonzaloriestra** [comment]
retest this please

**Comment by gonzaloriestra** [comment]
:heavy_check_mark: Acceptance OK by @cmongut 

But we'll wait to improve the error message from CARTOframes before merging: https://github.com/CartoDB/cartoframes/issues/1471

**Comment by oleurud** [comment]
I am going to make 2 changes here:
- I am going to change the quota status code error
- I am going to update the kuviz size limit to 10Mb

File: lib/carto/errors.rb [hunk 1]
```diff
@@ -84,4 +84,16 @@ def initialize(parameter, extra_message = nil)
...
   end
+
+  class PaymentRequiredError < CartoError
+    def initialize(message = "Payment Required")
+      super(message, 402)
+    end
+  end
+
+  class QuotaExceededError < PaymentRequiredError
+    def initialize(message = "Your quota has been exceeded. Please, upgrade your account")
```
Comment (Jesus89) [inline lib/carto/errors.rb]: "Your quota has been exceeded"

File: app/controllers/carto/api/public/custom_visualizations_controller.rb [hunk 2]
```diff
@@ -129,6 +131,11 @@ def check_edition_permission
...
   end
 
+  def check_public_map_quota
+    return unless CartoDB::QuotaChecker.new(@logged_user).will_be_over_public_map_quota?
+    raise Carto::QuotaExceededError.new('Public map quota exceeded')
+  end
+
   def validate_input_parameters
     if request.content_length > CONTENT_LENGTH_LIMIT_IN_BYTES
       return render_jsonp({ error: "visualization over the size limit (#{CONTENT_LENGTH_LIMIT_IN_BYTES})" }, 400)
```
Comment (Jesus89) [inline app/controllers/carto/api/public/custom_visualizations_controller.rb]: **V**isualization?

**Comment by gonzaloriestra** [comment]
@oleurud the changes LGTM :+1:

**Comment by oleurud** [comment]
retest this please

**Pull request merged**